lib/ocpp/v201/ctrlr_component_variables.cpp

```diff
@@ -13,12 +13,10 @@ const Component& AuthCacheCtrlr = {"AuthCacheCtrlr"};
 const Component& AuthCtrlr = {"AuthCtrlr"};
 const Component& ChargingStation = {"ChargingStation"};
 const Component& ClockCtrlr = {"ClockCtrlr"};
-const Component& Connector = {"Connector"};
 const Component& CustomizationCtrlr = {"CustomizationCtrlr"};
 const Component& DeviceDataCtrlr = {"DeviceDataCtrlr"};
 const Component& DisplayMessageCtrlr = {"DisplayMessageCtrlr"};
-const Component& EVSE = {"EVSE"};
-const Component& ISO15118Ctrlr = {"ISO15118Ctrlr"};
+const Component& ISO15118Ctrlr = {"ISO15118Ctrlr"}; // dynamic?
```

Comment:
dynamic?

Reply:
A question to self :D Will possibly need to be dynamic based on nr of evses in the future